src/facade/conn_context.h

```diff
@@ -107,6 +107,8 @@ class ConnectionContext {
   std::vector<uint64_t> acl_commands;
   // keys
   dfly::acl::AclKeys keys{{}, true};
+  // pub/sub
+  dfly::acl::AclPubSub pub_sub{{}, true};
```

Comment:
maybe we should consider in the future to not copy it for all connections but add shared data instances that are versioned

Reply:
That's not a bad idea, we are future complete now once this is merged so we can improve :)